client/src/auth/saga.js

```diff
@@ -38,10 +37,26 @@ export function* handleRegister({ payload: { history } }) {
   }
 }
 
+export function* handleRefreshActiveGames({ payload: { id } }) {
+  if (!id) {
+    yield put(alertActions.addAlert('undefined player ID', 'could not refresh active games'));
```

Comment:
Ope, `addAlert`'s second arg is en enum. We should define and export some constants to assist here, but these are the possible values:

https://github.com/joshprzybyszewski/cribbage/blob/892fbf53fcbdcd3d5adb6cc8c7920b5cc477faba/client/src/app/containers/Alert/index.jsx#L14-L25